files/image_config/dhcp_dos_logger/dhcp_dos_logger.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python3
+
+import re
+import os
+import subprocess
+import time
+from sonic_py_common.logger import Logger
+from swsscommon.swsscommon import ConfigDBConnector
+
+SYSLOG_IDENTIFIER = os.path.basename(__file__)
+
+# Global logger instance
+logger = Logger(SYSLOG_IDENTIFIER)
+logger.log_info("Starting DHCP DoS logger...")
+
+# Connect to config db
+config_db = ConfigDBConnector()
+config_db.connect()
+
+# Initialize
+drop_pkts = {}
+
+# Get list of ports
+ports = config_db.get_table('PORT').keys()
+
+# Initialize the ports with zero initial packet drops
+drop_pkts = {port: 0 for port in ports}
+
+# Main handler function
+def handler():
+    """
```

Comment:
Could you add test case and check test coverage?

Reply:
test coverage required for handler? currently, as per practice we don't add test cases and coverage here. The same is followed by all other features in image_config.